doc/modules/preprocessing.rst

```diff
@@ -698,16 +688,13 @@ It is possible to adjust the threshold of the binarizer::
          [1., 0., 0.],
          [0., 0., 0.]])
 
-As for the :class:`StandardScaler` and :class:`Normalizer` classes, the
-preprocessing module provides a companion function :func:`binarize`
-to be used when the transformer API is not necessary.
 
 Note that the :class:`Binarizer` is similar to the :class:`KBinsDiscretizer`
 when ``k = 2``, and when the bin edge is at the value ``threshold``.
 
 .. topic:: Sparse input
 
-  :func:`binarize` and :class:`Binarizer` accept **both dense array-like
+  :class:`Binarizer` accepts **both dense array-like
```

Comment:
For the same reason I think we should revert this part and still mention `binarize`

Reply:
Ok